src/logger.coffee

```diff
@@ -209,7 +186,7 @@ module.exports = class Logger
 			if stdoutOrStderr not in [ 'stdout', 'stderr' ]
 				throw new Error("Invalid log selection #{stdoutOrStderr}")
 			if !@attached[stdoutOrStderr][containerId]
-				logsOpts = { follow: true, stdout: stdoutOrStderr == 'stdout', stderr: stdoutOrStderr == 'stderr', timestamps: true }
+				logsOpts = { follow: true, stdout: stdoutOrStderr == 'stdout', stderr: stdoutOrStderr == 'stderr', timestamps: true, since: '0s' }
```

Comment:
This means that we'll lose any logs from the application that happen on boot before the supervisor starts will be lost, which at the very least should be flagged up to support agents as I wouldn't be surprised if it comes up

Reply:
Isn't `0` the default? Looking at https://docs.docker.com/engine/api/v1.29/#operation/ContainerLogs
(and it should be an integer, not a string?)

Also, (I don't see how this would happen here, but) do we want to lose logs from before the supervisor starts? This used to be an issue that users noted/reported often until it was fixed [here](h.ttps://github.com/resin-io/resin-supervisor/pull/394)

Reply:
@pcarranzav for some reason it worked correctly with a recent docker but didn't work with older versions. I changed it to `Math.floor(Date.now() / 1000)`

Reply:
@petrosagg but are you sure we want to bring this behavior back?

This was done in this way (getting all logs) on purpose, because otherwise the first logs are always lost. Getting only the logs after supervisor start only fixes the case of a supervisor restart (which is rare, except on support actions and on supervisor updates) but reintroduces a known issue for all other cases.

Or is there something else I'm missing?